recipe/meta.yaml

```diff
@@ -21,7 +21,7 @@ requirements:
   - bob.extension
   - bob.blitz
   - bob.core
-  - hdf5
+  - hdf5 1.8.15*
```

Comment:
> OK, that is good to know. I noticed that continuum pin to the patch version, but sounds like you think we can patch to the minor version? Out of interest, how do you know?

After a lot of pain! The packages: `hdf4`, `hdf5`, `h5py`, `h5netcdf`, and `netcdf` are essential for my day job.

In the `IOOS` channel I avoided having my own version of those packages and relied on defaults to avoid problems. But in conda-forge @jakirkham needed a newer `hdf5` and `h5py` breaking any env that uses `netcdf`. (You'll notice I pinned `hdf5` in many places because of that.) That gave me most of my packages back, but I still cannot install `h5py` with the older `hdf5` (b/c of `libnetcdf`).

Long story short... We really need to get our own `libnetcdf` ASAP.